website/settings/defaults.py

```diff
@@ -1831,4 +1831,5 @@ def parent_dir(path):
     'bluebook-inline': 'bluebook'
 }
 
-PREPRINTS_ASSETS = '/static/img/preprints_assets/'
+OSF_CDN_DOMAIN = 'https://staging-cdn.osf.io'
```

Comment:
Correct. For production you'll use `https://osf.io/preprints/assets/osf-assets/files/preprints-assets/<id>/square_color_no_transparent.png` instead.